templates/group_vars_mash_servers

```diff
@@ -1481,6 +1486,43 @@ changedetection_container_labels_traefik_tls_certResolver: "{{ devture_traefik_c
 
 
 
+# role-specific:wetty
+########################################################################
+#                                                                      #
+# wetty                                                                #
+#                                                                      #
+########################################################################
+
+wetty_enabled: false
+
+wetty_identifier: "{{ mash_playbook_service_identifier_prefix }}wetty"
+
+wetty_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}wetty"
+
+wetty_uid: "{{ mash_playbook_uid }}"
+wetty_gid: "{{ mash_playbook_gid }}"
+
+wetty_container_additional_networks_auto: |
+  {{
+    ([mash_playbook_reverse_proxyable_services_additional_network] if mash_playbook_reverse_proxyable_services_additional_network else [])
+  }}
+
+# role-specific:traefik
+wetty_container_labels_traefik_enabled: "{{ mash_playbook_traefik_labels_enabled }}"
```

Comment:
Using `role-specific` is generally a good idea, but here it might cause problems.

If the `traefik` role is not in use, then all these lines may be removed and `wetty_container_labels_traefik_enabled` will not be set in terms of `mash_playbook_traefik_labels_enabled`.

The role default for `wetty_container_labels_traefik_enabled` is also `true`: https://github.com/spatterIight/ansible-role-wetty/blob/773cd1b53f5a07e974a9f12ae64c1a5834879df3/defaults/main.yml#L48

.. so labels will be added, even if the user doesn't want them.

----

I suppose that the variables which depend on `devture_traefik_` variables may be put in a `role-specific` block, but all others (`wetty_container_labels_traefik_enabled` and `wetty_container_labels_traefik_docker_network`) should probably remain outside of it.

Reply:
Went ahead and move those two variables (`wetty_container_labels_traefik_enabled` and `wetty_container_labels_traefik_docker_network`) outside of the `role-specific` block

However, I am confused since many other roles do not follow this pattern

![image](https://github.com/mother-of-all-self-hosting/mash-playbook/assets/81454789/eff66633-544c-490d-86fd-5363723a9610)

Additionally, I should change the role default of `wetty_container_labels_traefik_enabled`  to `false`, correct?

Reply:
Good point! I have adjusted all the others to follow the same pattern in e3e2aa4fb8b0d24bbfc41d5c22dbbd08811a6326

I think you can leave the role default to `true`. All other roles do the same.